### ISSUE TITLE
syringe

### ISSUE THREAD

**Issue by veersamrat** [Templarian/MaterialDesign#1203]
Can you add medical related icons like tablets, syrups, drops etc..

**Comment by CoDEmanX** [comment]
Not very medical: ![candy](https://cloud.githubusercontent.com/assets/1309249/15009362/41665d34-11e6-11e6-83df-283b4ab5b83d.png)

**Comment by Templarian** [comment]
@CoDEmanX I know it's suppose to be DNA, but it looks like candy.

**Comment by CoDEmanX** [comment]
It's actually a candy as icon for "drop" xD
Looks more like a casual candy than a medical sucking sweets / hard candy I admit.

**Comment by jacobq** [comment]
I just created a syringe icon for my application but don't have a full set of 5 to contribute. Perhaps someone else (who might actually be a designer :D) can take it, improve it, and include it with the medical icons.
![syringe-192](https://cloud.githubusercontent.com/assets/1633459/15077187/e9db2f2a-1372-11e6-8de1-c7b179577c18.png)
[syringe.zip](https://github.com/Templarian/MaterialDesign/files/252528/syringe.zip)

**Comment by CoDEmanX** [comment]
There's already a needle, but I assume the semantics are different?
https://materialdesignicons.com/icon/needle

**Comment by jacobq** [comment]
Thanks for pointing that out. Actually, I did not know about that other icon or I wouldn't have taken the time to make this one. I didn't scan through the entire collection and didn't think of searching for 'needle'. (I tried things like syringe, inject, immunization, inoculation, etc. because I thought it might be associated with medical things but did not try 'needle', 'plunger', etc. because I wasn't thinking about its constituent parts. )
In my case, the application has nothing to do with the medical industry; it is a chemistry application where a robotic system uses a syringe to extract samples of liquid from vials and slowly inject it into an instrument for analysis. Now that there is a nice test tube icon (https://github.com/Templarian/MaterialDesign/issues/1209) I may use that instead as I think it better represents the idea of a substance to be analyzed. 

![autosampler](https://cloud.githubusercontent.com/assets/1633459/15113646/273adbc8-15bb-11e6-80d2-de7d166cc4c1.png)
